cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ParameterUtils.java

```diff
@@ -408,16 +409,31 @@ static Long time(HttpServletRequest request) {
     return timeString.toUpperCase().equals("NOW") ? System.currentTimeMillis() : Long.parseLong(timeString);
   }
 
-  static long startMs(HttpServletRequest request) {
-    String parameterString = caseSensitiveParameterName(request.getParameterMap(), START_MS_PARAM);
-    return parameterString == null ? DEFAULT_START_TIME_FOR_CLUSTER_MODEL
-                                   : Long.parseLong(request.getParameter(parameterString));
+  @Nullable
+  static Long startMs(HttpServletRequest request) {
+    return parseParamAsLong(request, START_MS_PARAM);
   }
 
-  static long endMs(HttpServletRequest request) {
-    String parameterString = caseSensitiveParameterName(request.getParameterMap(), END_MS_PARAM);
-    return parameterString == null ? System.currentTimeMillis()
-                                   : Long.parseLong(request.getParameter(parameterString));
+  @Nullable
+  static Long endMs(HttpServletRequest request) {
+    return parseParamAsLong(request, END_MS_PARAM);
+  }
+
+  static long startMsOrDefault(HttpServletRequest request, long defaultStartMs) {
+    Long startMs = parseParamAsLong(request, START_MS_PARAM);
+    return startMs == null ? defaultStartMs : startMs;
+  }
+
+  static long endMsOrDefault(HttpServletRequest request, long defaultEndMs) {
```

Comment:
1. For consistency of the parameter name for missing default values `defaultEndMs` -> `defaultIfMissing`. 
2. Also, can we make the return type and the type of the default parameter `Long`? Clearly, if the response is `null`, it would only be possible via a `null` default parameter passed to this function.

Reply:
OK, that makes sense